.github/workflows/scheduled_build.yml

```diff
@@ -0,0 +1,48 @@
+name: Hourly build
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  build:
+    if: (github.event_name == 'schedule' && github.repository == 'Roave/SecurityAdvisoriesBuilder') || (github.event_name != 'schedule')
```

Comment:
`&& github.repository == 'Roave/SecurityAdvisoriesBuilder'` is this still needed at all?

Reply:
Also: why are other events considered? Maybe we want to only allow a manual trigger?

Reply:
Thanks for clarifying!

I think we should note this in a comment in here, to make the conditional clear.

Reply:
I'm going to add it in next PR